### DISCUSSION TITLE
Too much power!

### DISCUSSION THREAD

**Discussion by Grey-Sheep** [springfall2008/batpred#1333] [Q&A]
I have a 9Kw export limit on my installation, and Givenergy have confirmed the gateway is configured for that export value.

Today (and several times in the past) I have noticed I am exporting at 12Kw as shown below
![12KVA export](https://github.com/user-attachments/assets/3a5444c7-2d5d-473d-b3ca-b71cbf21312e)

I have the export level in apps.yaml set to 9000, but this is for prediction calculations.
I have the one inverter set at 6000 for the AIO

But today, the AIO and Panels are exporting everything they can and that is 12Kw.

I must have missed a setting🤦

Could anyone point me in the right direction.

Thanks in anticipation

**Comment by gcoan** [comment]
Predbat works out the predicted export, taking into account your export limit.  It doesn't control the actual inverter export, that's down to the inverter.

If GE says it is correctly configured then practically you've done all you can.
I would doubt the DNO would actually ever notice.  Enjoy the extra export income

**Comment by cyberkryten** [comment]
Is this a brief '12kW' export and then back to 9kW or is it sustained at far over the limit?

I have had a similar issue (SE), the issue is entirely with the inverter config/control.  You should really have had G100 limits kick in and shut it down, so either that isn't supported or configured - though I guess you have an AC coupled battery and a separate inverter - if so then they have to communicate to decide who gets to export what as if both are set to max of 6kW then clearly that's going to be over the limit - that's an installer/GE question.

I don't know how/when/if DNOs check things, but to be safe it is worth checking what the AC voltage is at the meter as export restrictions can be related to problems controlling max voltage and if it is spiking outside of limits then that's not good for anyone.

**Comment by Grey-Sheep** [comment]
This was a sustained export!

![image](https://github.com/user-attachments/assets/8dec8baa-2ddd-492c-98e8-384875960979)
